upgrade_zookeeper.yml

```diff
@@ -19,6 +19,9 @@
       import_role:
         name: confluent.zookeeper
         tasks_from: health_check.yml
+      when:
+        - health_checks_enabled|bool
```

Comment:
remove the health_checks_enabled flag, upgrades should not be able to skip for that reason